flask/documentation/get_namespace_admins.yaml

```diff
@@ -1,18 +1,26 @@
-description: Get admins of a namespace.
+description: Verifies by uuid of the user and gets the namespace admins.
+
 parameters:
   - name: namespace
     in: path
     description: Namespace name
     required: true
     type: string
+
+  - name: uuid
+    in: formData
+    required: true
+    type: string
+    description: UUID of the user
```

Comment:
Thanks @minhqdao for the feedback. Yes, this information is not to be entered by the registry user. And we are not in any case allowing user to manually enter it from frontend. 

This is automatically appended in code. 

But, since this API & it's documentation is for developers, they will have to provide it inside the form field in playground for testing.  

https://github.com/fortran-lang/registry/pull/46/files#diff-17ad28ce3f853ac32e416110318d70bbe11f5a4fe10cff1f231bee19bd64faabR19

Reply:
@minhqdao Should I consider this as resolved ?